src/popup/router/components/Filters.vue

```diff
@@ -35,15 +35,28 @@ export default {
   props: {
     value: { type: Object, required: true },
     filters: { type: Object, required: true },
-    fixed: Boolean,
+    sticky: Boolean,
+    withSearch: Boolean,
   },
   data() {
     return {
       rotatableFilters: this.filters,
     };
   },
+  computed: {
+    cssVars() {
+      return this.withSearch ? {
+        '--header-height': '116px',
+        '--account-info-height': '62px',
```

Comment:
This makes this component dependant on the fact that the parent component is the AccountInfo. In my opinion components should be parent-agnostic.